var/spack/repos/builtin/packages/flang/package.py

```diff
@@ -13,32 +13,47 @@ class Flang(CMakePackage):
 
     homepage = "https://github.com/flang-compiler/flang"
 
-    url      = "https://github.com/flang-compiler/flang/archive/flang_20180612.tar.gz"
+    url      = "https://github.com/flang-compiler/flang/archive/flang_20190329.tar.gz"
     git      = "https://github.com/flang-compiler/flang.git"
 
-    version('develop', branch='master')
+    maintainers = ['naromero77']
+
+    version('master', branch='master')
+    version('20190329', sha256='b8c621da53829f8c53bad73125556fb1839c9056d713433b05741f7e445199f2')
+    version('20181226', sha256='00e716bea258c3bb60d6a5bb0c82bc79f67000062dc89939693e75f501883c36')
     version('20180921', sha256='f33bd1f054e474f1e8a204bb6f78d42f8f6ecf7a894fdddc3999f7c272350784')
     version('20180612', sha256='6af858bea013548e091371a97726ac784edbd4ff876222575eaae48a3c2920ed')
 
-    depends_on('llvm@develop+flang~gold~lldb~lld~compiler-rt~libcxx~polly~internal_unwind', when='@develop')
-    depends_on('llvm@7.0.1+flang~gold~lldb~lld~compiler-rt~libcxx~polly~internal_unwind', when='@20180921')
-    depends_on('llvm@6.0.0+flang~gold~lldb~lld~compiler-rt~libcxx~polly~internal_unwind', when='@20180612')
+    # Build dependency
+    depends_on('cmake@3.8:', type='build')
+    depends_on('python@2.7:', type='build')
+
+    depends_on('llvm-flang@release_70', when='@master')
+    depends_on('llvm-flang@20190329', when='@20190329')
+    depends_on('llvm-flang@20181226_70', when='@20181226')
+    depends_on('llvm-flang@20180921', when='@20180921')
+    depends_on('llvm-flang@20180612', when='@20180612')
 
-    depends_on('pgmath@develop', when='@develop')
+    depends_on('pgmath@master', when='@master')
+    depends_on('pgmath@20190329', when='@20190329')
+    depends_on('pgmath@20181226', when='@20181226')
     depends_on('pgmath@20180921', when='@20180921')
     depends_on('pgmath@20180612', when='@20180612')
 
     def cmake_args(self):
+        spec = self.spec
         options = [
             '-DWITH_WERROR=OFF',
             '-DCMAKE_C_COMPILER=%s' % os.path.join(
-                self.spec['llvm'].prefix.bin, 'clang'),
+                spec['llvm-flang'].prefix.bin, 'clang'),
             '-DCMAKE_CXX_COMPILER=%s' % os.path.join(
-                self.spec['llvm'].prefix.bin, 'clang++'),
+                spec['llvm-flang'].prefix.bin, 'clang++'),
             '-DCMAKE_Fortran_COMPILER=%s' % os.path.join(
-                self.spec['llvm'].prefix.bin, 'flang'),
+                spec['llvm-flang'].prefix.bin, 'flang'),
             '-DFLANG_LIBOMP=%s' % find_libraries(
-                'libomp', root=self.spec['llvm'].prefix.lib)
+                'libomp', root=spec['llvm-flang'].prefix.lib),
+            '-DPYTHON_EXECUTABLE={0}'.format(
+                os.path.join(spec['python'].prefix.bin, 'python'))
```

Comment:
```suggestion
                spec['python'].command.path)
```
In Python 3, the `python` executable may not exist and only `python3` is present.

Reply:
will fix